resumes/views.py

```diff
@@ -1,2 +1,32 @@
 from django.views       import View
 from django.http        import JsonResponse
+
+from resumes.models     import Resume
+from utils              import authorization
+
+class ResumeDetailView(View):
+    @authorization
+    def get(self, request, resume_id):
+        try:
+            resume = Resume.objects.get(id=resume_id)
+
+            if resume.user != request.user:
+                return JsonResponse({"message" : "USER_NOT_MATCH"}, status=401)
```

Comment:
```suggestion  
            if Resume.objects.filter(id=resume_id, user=request.user).exists():
                return JsonResponse({"message" : "USER_NOT_MATCH"}, status=401)
            
            resume = Resume.objects.get(id=resume_id, user=request.user)
```
이력서가 특정 유저의 의존하고 있는 자원이기 때문에 유저와 같이 존재 여부를 판단하게 되면, 이력서의 존재 유무, 해당 유저의 이력인가의 대한 유무를 동시의 판단이 가능합니다.

Reply:
넵! 수정완료했습니다!